### DISCUSSION TITLE
[Miscellaneous]  bridgeERC20to  tokens not received , BOUNTY

### DISCUSSION THREAD

**Discussion by foobar289** [ethereum-optimism/developers#380] [Miscellaneous]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

hi , i have a problem and willing to offer bounty for this , i bridged wsteth to base calling bridgeERC20to , but i have not received my tokens on base,  i am aware base and optimism uses the same bridge, is there anything i could do to get my tokens? 
this is my ethereum tx https://etherscan.io/tx/0xa558b594235c7652cd7f233734b68c5ea68f74d1dbd15e381ab076531247992f

**Comment by hash1go** [comment]
I saw your question on the Discord channel as well. Please try issuing the following transaction to attempt the message relay again.

1. Visit:
https://basescan.org/address/0x4200000000000000000000000000000000000007#writeProxyContract
2. Open relayMessasge tab
3. Connect your wallet
4. Input follows:
relayMessage
`0`
_nonce (uint256)
`1766847064778384329583297500742918515827483896875618958121606201292786044`
_sender (address)
`0x3154Cf16ccdb4C6d922629664174b904d80F2C35`
_target (address)
`0x4200000000000000000000000000000000000010`
_value (uint256)
`0`
_minGasLimit (uint256)
`132303`
_message (bytes)
`0x0166A07A000000000000000000000000C1CBA3FCEA344F92D9239C08C0568F6F2F0EE4520000000000000000000000007F39C581F595B53C5CB19BD0B3F8DA6C935E2CA00000000000000000000000006FCE63859A859A0F30ED09B12F5010D790618CA40000000000000000000000006FCE63859A859A0F30ED09B12F5010D790618CA400000000000000000000000000000000000000000000000029B1B962235BD63700000000000000000000000000000000000000000000000000000000000000C00000000000000000000000000000000000000000000000000000000000000000`

5. Click write button


Since I can't simulate transactions in my environment, I cannot confirm that you will receive wstETH with this transaction. If you are using Rabby Wallet, you should be able to detect changes in your account balance as a result of executing the transaction, so please make sure to check before proceeding.
I cannot guarantee compensation if assets are permanently lost due to that transaction.

**Reply by linkeong88** [reply]
This doesn’t work, you can see he attempted this before but it returns the same FailedRelayedMessage. There is a problem at the contract level at the bridge

https://basescan.org/tx/0xab75fd782ca9cbeee316e6fa5f679d90efeabfc47b22d23e5e8f46a879071e46#eventlog

**Reply by hash1go** [reply]
@foobar289 @linkeong88 
Could you provide the website where this deposit transaction was issued, along with the specific steps?

From my investigation, it appears that issuing a deposit transaction for wstETH from bridge.base.org/deposit may not correctly bridge the assets.

**Reply by linkeong88** [reply]
Yes thank you the bridge site doesn’t show so the only way was to interact with the contract either directly or through this

https://etherscan.io/address/0x3154cf16ccdb4c6d922629664174b904d80f2c35#writeProxyContract

**Reply by linkeong88** [reply]
https://docs.optimism.io/builders/app-developers/bridging/standard-bridge

**Reply by hash1go** [reply]
Hello. I apologize for the delay in providing an update on this matter.
I checked with the engineer at OP Labs, and it seems that your tokens are stuck and cannot be recovered.

**Comment by linkeong88** [comment]
I would hope that the dev actually try to see if there are any solutions rather than just brush it off, as the tokens are now held by the chain owners and anyone bridging to the chain had trusted the credibility of the devs, this will take a big hit on the credibility and drive away users. Furthermore this is not an isolated case or edge case that devs are trying to paint as can be seen from previous postings and other people getting stuck
